x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/integration/MlDistributedFailureIT.java

```diff
@@ -127,15 +124,9 @@ public void testFullClusterRestart() throws Exception {
     public void testCloseUnassignedJobAndDatafeed() throws Exception {
         internalCluster().ensureAtMostNumDataNodes(0);
         logger.info("Starting dedicated master node...");
-        internalCluster().startNode(Settings.builder()
-                .put("node.master", true)
-                .put("node.data", true)
-                .put("node.ml", false)
-                .build());
+        internalCluster().startMasterOnlyNode();
```

Comment:
The code used to enable `master` and `data` here, was that a mistake in the old code?

Reply:
It looks like the logging message is what is incorrect here, and the test indeed needs two data nodes.